apps/server/src/shared/infra/antivirus/antivirus.service.ts

```diff
@@ -1,7 +1,10 @@
 import { AmqpConnection } from '@golevelup/nestjs-rabbitmq';
 import { Inject, Injectable, InternalServerErrorException } from '@nestjs/common';
 import { ErrorUtils } from '@src/core/error/utils';
+import { ScanResultParams } from '@src/modules/files-storage/controller/dto';
```

Comment:
Can we places it in this module and use in filesstorage. The dependency order is wrong.